services/apps/alcs/src/alcs/application-decision/application-boundary-amendment/application-boundary-amendment.entity.ts

```diff
@@ -52,6 +52,15 @@ export class ApplicationBoundaryAmendment extends Base {
   })
   period: number;
 
+  @Column({
+    select: false,
+    nullable: true,
+    type: 'int8',
+    comment:
+      'This column is NOT related to any functionality in ALCS. It is only used for ETL and backtracking of imported data from OATS. It links oats.oats_components to alcs.application_boundary_condition.',
```

Comment:
It does not look like that OATS has oats.oats_components. Did you mean oats.oats_alr_appl_components

Reply:
yep, that's correct